library/src/scala/compiletime/ops/package.scala

```diff
@@ -6,39 +6,33 @@ package object ops {
   @infix type ==[X <: AnyVal, Y <: AnyVal] <: Boolean
```

Comment:
I believe `==` and `!=` should be split as well. Either everything is split according to the supertype or non of it is.

Reply:
I moved them into `scala.compiletime.ops.any` in 8c117c1342913a8d23cf59cd2321afbbecc7e952.

The alternative was to duplicate them into each subpackage for each supported type, and then duplicate the constant folding code... which felt like a lot of duplication. Seeing that `==` and `!=` are [defined on `Any`](https://www.scala-lang.org/api/2.13.1/scala/Any.html#!=(x$1:Any):Boolean), I think this solution makes the most sense. It also emphasizes that equality is between to `Any` values, and that `1 == "1"` will return `false`.